worlds/witness/options.py

```diff
@@ -279,12 +290,31 @@ class ChallengeLasers(Range):
     default = 11
 
 
-class ElevatorsComeToYou(Toggle):
+class ElevatorsComeToYou(OptionSet):

```

Comment:
It may be a good idea to add the list of valid keys to the option description, this makes it clear what the options are even in the template yaml.